src/target/stm32f4.c

```diff
@@ -99,6 +110,11 @@
 #define AXIM_BASE 0x8000000U
 #define ITCM_BASE 0x0200000U
 
+#define STM32F7_ITCM_RAM_BASE 0x0U
```

Comment:
Do note this should really have all 8 digits present for consistency please - we know they're all 0's but this at least makes the constants all follow the same form.

Reply:
Amended and fast-forwarded.